source/geh_common/release-notes.md

```diff
@@ -1,5 +1,9 @@
 # GEH Common Release Notes
 
+## Version 5.6.2
+
+- changed spark config to remove hive configurations and reduce unnesseray noise in test output.
```

Comment:
This, the title, and the description need to be updated (hive is no longer part of this PR).